lambdas/credentialissuerreturn/src/main/java/uk/gov/di/ipv/core/credentialissuerreturn/CredentialIssuerReturnHandler.java

```diff
@@ -66,11 +76,17 @@ public APIGatewayProxyResponseEvent handleRequest(
                             accessToken, credentialIssuerConfig, request.getIpvSessionId());
             credentialIssuerService.persistUserCredentials(verifiableCredential, request);
 
+            sqsService.sendAuditEvent(
+                    AuditEventTypes.IPV_CREDENTIAL_RECEIVED_AND_SIGNATURE_CHECKED);
+
```

Comment:
Yeah I was a little unsure of wether it should be before or after the persistence. I went with after as I thought it could be hard to follow if we've got an audit trail saying that we "received" a VC but then there being no record of it in the DB if there were DynamoDb issues.

But I see your point as well so I'm happy to change this

Reply:
Yeah it's a funny one. I would hope that if something did happen with the DB though that we'd catch an error and log it, so we should be able to work out what happened.